app/models/site_config.rb

```diff
@@ -89,7 +89,7 @@ class SiteConfig < RailsSettings::Base
   # Images
   field :main_social_image, type: :string, default: proc { URL.local_image("social-media-cover.png") }
 
-  field :favicon_url, type: :string, default: "favicon.ico"
+  field :favicon_url, type: :string, default: proc { URL.local_image("favicon.ico") }
```

Comment:
Unless we wanted to purposely avoid providing a default favicon, I updated it to provide the DEV favicon as the default favicon.